internal/pkg/build/buildkit/client/client.go

```diff
@@ -255,7 +257,7 @@ func newSolveOpt(_ context.Context, opts *Opts, w io.WriteCloser, buildDir, spec
 		spec = filepath.Join(buildDir, "Dockerfile")
 	}
 	localDirs := map[string]string{
-		"context":    buildDir,
+		"context":    "/",
```

Comment:
This isn't correct for Dockerfile builds.

In Docker an `ADD` cannot add files outside of the context, and the context is that path specified on the command line.

See - https://docs.docker.com/build/building/context/

Reply:
Right, ok – so we should probably discuss this. The previous value, `buildDir`, made it so that basically nothing could be ADDed, since it pointed to the temporary directory that didn't even exist prior to the build.

We could perhaps make it point to the current working directory by default, and perhaps wire up the `--cwd(/--pwd)` flag to the `build` command (in OCI mode only) to override this if the user wants to?

(Docker builds from Dockerfile, of course, typically involve an explicit directory argument, so this doesn't arise.)